test/e2e/main_test.go

```diff
@@ -260,69 +260,6 @@ func TestCacheService(t *testing.T) {
 	}
 }
 
-// TestPermanentCache creates a permanent caches the stop/start
-// the cluster and checks that the cache is still there
-func TestPermanentCache(t *testing.T) {
```

Comment:
Are we deleting the test case as it's duplication of `TestCheckDataSurviveToShutdown`?

Reply:
> Are we deleting the test case as it's duplication of `TestCheckDataSurviveToShutdown`?

actually no, it tests permanent cache... let me see if I can fix it, instead of removing.
thanks @Crumby

Reply:
@Crumby should be ok now